docs/Basic-Tutorial.md

```diff
@@ -139,10 +139,12 @@ with your local OpenTripPlanner instance.
 
 This map-based user interface is in fact sending HTTP GET requests to the OTP server running on your
 local machine. It can be informative to watch the HTTP requests and responses being generated using
-the developer tools in your web browser. OTP's built-in web server will run by default on ports 8080
-and 8081 for HTTP and HTTPS respectively. If by any chance some other software is already using one
-or both of those port numbers, you can specify different port numbers with switches
-like `--port 8801 --securePort 8802`.
+```suggestion

```

Comment:
```suggestion
```
This messes up rendering.

Reply:
I have no idea how I or github managed to add that there :D.